apps/researcher/src/app/[locale]/objects/initial-search-field.tsx

```diff
@@ -0,0 +1,17 @@
+'use client';
+
+import {SearchFieldWithLabel} from '@colonial-collections/ui/list';
+import {useRouter} from '@/navigation';
+
+export function InitialSearchField() {
+  const router = useRouter();
+
+  const navigateOnSearch = (query: string) => {
+    if (query) {
```

Comment:
To be sure: does this check rule out a `query` such as `0`? If so, should the check be: `if (query.length > 0)`?